authentication/utils.py

```diff
@@ -0,0 +1,27 @@
+from firebase_admin import auth
+from rest_framework.status import HTTP_422_UNPROCESSABLE_ENTITY, HTTP_400_BAD_REQUEST
+from rest_framework.exceptions import ValidationError
+
+class FirebaseAPI:
+    @classmethod
+    def verify_id_token(cls, id_token):
+        try:
+            decoded_token = auth.verify_id_token(id_token)
+            return decoded_token
+        except ValueError:
+            raise ValidationError('Invalid Firebase ID Token.', HTTP_422_UNPROCESSABLE_ENTITY)
```

Comment:
We have a try-except block here and in the serializers.py file as well. Can't we just do it once?